octopusdeploy/schema_team.go

```diff
@@ -71,20 +71,15 @@ func flattenTeam(team *octopusdeploy.Team) map[string]interface{} {
 func getTeamDataSchema() map[string]*schema.Schema {
 	dataSchema := getTeamSchema()
 	setDataSchema(&dataSchema)
+	delete(dataSchema, "user_role")
 
 	return map[string]*schema.Schema{
 		"id":             getDataSchemaID(),
 		"ids":            getQueryIDs(),
 		"include_system": getQueryIncludeSystem(),
 		"partial_name":   getQueryPartialName(),
 		"skip":           getQuerySkip(),
-		"spaces": {
```

Comment:
Why is this being removed? It's a filter available on the `teams` endpoint.

Reply:
It was removed because it looked like a copy paste error. The schema for the space filter was set as a list of team objects. I'll look up what the endpoint expects and add it back.

Reply:
FWIW, here's the URI template for the `teams` endpoint:

`/api/scopeduserroles{/id}{?skip,take,ids,partialName,spaces,includeSystem}`